app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -1,6 +1,16 @@
 angular.module('miq.util').factory('chartsMixin', function() {
   'use strict';
 
+  var hourlyTimeTooltip = function(d) {
+    var theMoment = moment(d[0].x);
+    return _.template('<table class="c3-tooltip">' +
```

Comment:
btw where is the tooltip styling coming from? I think the tooltip should look slightly different:
https://www.patternfly.org/patterns/utilization-bar-chart/
(notice that it has a triangle/arrow from its "source")